metadata/constants.go

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metadata
+
+// Common constants
+const (
+	// DefaultReleaseGracePeriodDays is the default time to live in days of a Release
+	DefaultReleaseGracePeriodDays = 7
```

Comment:
@davidmogar do we want this here? Is it going to be in the ReleaseServiceConfig? Can we just use kubebuilder defaults instead of having the constant?

Reply:
No, it shouldn't be here. As you said, and just to give some more context to Leandro, defaults will be moved in the future (different issue) to the ReleaseServiceConfig resource. For the time being, using a kubebuilder annotation is a much better and cleaner solution. You can find it [here](https://book.kubebuilder.io/reference/markers/crd-validation) in case you didn't know about it.

Reply:
file removed.